toplevel/genprintval.ml

```diff
@@ -270,7 +274,7 @@ module Make(O : OBJ)(EVP : EVALPATH with type valu = O.t) = struct
       let nested_values = ObjTbl.create 8 in
       let nest_gen err f depth obj ty =
         let repr = obj in
-        if not (O.is_block repr) then
```

Comment:
It's a bit annoying that `Null` is considered a block; maybe this file would be easier to read by defining `let is_real_block o = O.is_block o && not (is_null o)` ?